account-integrations/sdk/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@getwax/sdk",
+  "version": "1.0.0",
```

Comment:
Don't know if this matters too much but since this is early doors on the sdk, would we consider starting on a version number like `0.1.0` to indicate it's development stage?

Reply:
yeah, good call.  I'll actually change it to 0.0.1